Makefile

```diff
@@ -0,0 +1,99 @@
+CABALS := \
```

Comment:
I agree that we should have a script to run hpack on `package.yaml` files, but this seems like it would be simpler as a shell strict.

Reply:
Yes, maybe we will forget to add new cabal files here.